content/en/blog/2023/announcing-major-release.md

```diff
@@ -0,0 +1,93 @@
+
+- Verify signatures using [trust store and trust policy](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/trust-store-trust-policy.md) with fine-tuned configuration for repository specific, and multiple signature verification levels and multiple signature verification levels.  [notation policy](https://notaryproject.dev/docs/cli-reference/notation_policy/) command is also introduced in v1 release to simplify the policy management
+
+### Enhanced debug and troubleshooting experience
+
+[notation inspect](https://notaryproject.dev/docs/cli-reference/notation_inspect/) command is available to get detailed information of signatures associated with the signed artifact in a human readable view.
+
+In addition, `--verbose` flag with all CLI commands, providing troubleshooting capability for developers and users respectively.
+
+### Experimental features
+
+Experimental features are intended for testing and evaluation purposes only and should not be used in production environments. Users can enable experimental features in Notation CLI by setting the environment variable `NOTATION_EXPERIMENTAL` to 1. 
+
+There are two major features which are marked as experimental stage but might be useful in some specific scenarios.
+ 
+- Support [signing, listing and verification with OCI image layout](https://notaryproject.dev/docs/how-to/oci-image-layout/) for signing images before they are pushed to a registry. It enables users to sign and verify images in an air-gapped environment.
+- Support [OCI distribution referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc2/spec.md#enabling-the-referrers-api), which allows Notation CLI to fetch a list of referrers in an efficient and clean manner.
+
+### New specifications
+
+[Notary Project specifications](https://github.com/notaryproject/specifications/releases/tag/v1.0.0) reach its first major release. All specifications, requirements, scenarios, threat model, and security audit reports are available in this release. To interoperate with Notary Project tooling or develop your own client implementation, follow the latest Notary Project specifications for details.
+
+### Better extensibility and more plugins support
+
+Notation has an [extensible plugin framework design](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/plugin-extensibility.md) which provides plugin interfaces for users and vendors to implement their own integrations with the key/certificate management solutions or signing service. Notation has the following plugins available to use now.
+
+- [AWS Signer plugin](https://docs.aws.amazon.com/signer/latest/developerguide/Welcome.html)
+- [Azure Key Vault plugin](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-tutorial-sign-build-push)
+- [HashiCorp Vault plugin](https://github.com/notaryproject/notation-hashicorp-vault) (Alpha)
```

Comment:
Suggest removing this plugin as not released, and considering recent license change.

Reply:
I changed the status of the HashiCorp Vault plugin to `Experimental`, which means it is still in development and for testing purposes. Considering this is an open-source popular KMS, I think we should at least provide visibility for users especially who want an on-prem KMS solution with Notation. 

For the license change, I think it doesn't impact the HashiCorp Vault plugin since this plugin only uses Vault's API. @justincormack  has a confirmation on https://github.com/cncf/foundation/issues/617#issuecomment-1675803976

Reply:
Another consideration is to move the HashiCorp Vault plugin to the "What's the Next" section. It will reduce concerns here.

Reply:
After a discussion with @yizha1 . I will follow the second option as mentioned above.